client/styles/styles.css

```diff
@@ -270,6 +270,92 @@ body {
   color: #ffffff;
 }
 
+/* CAROUSEL-COMPONENT */
+
+.homepage__carousel {
+  background-color: #ffffff;
+  text-align: center;
```

Comment:
I would add a minimum padding
```
  padding-left: 80px;
  padding-right: 80px;
```

Reply:
I'll try that again, but last night I tried some extra paddings, but they didn't wok, again, It has something to do with the image size I think.

Reply:
It does not work @dirk-jacobs. You know, the images are inside 3 different <div>, I would have to try with each one. Or maybe resize the images with CSS, which I tried with no success, or resize the images outside the project and put it back in the image's folder. That's why I pushed it like this and move on to Pojects Page.

Reply:
Did you added it to the correct place ?
In my environment it worked